sdk/storage/azure-storage-queue/azure/storage/queue/aio/_models.py

```diff
@@ -15,16 +17,37 @@
 from .._models import QueueMessage, QueueProperties
 
 
+async def async_queue_msg_generator(messages: Any):
+        for q in messages:
+            yield QueueMessage._from_generated(q)
```

Comment:
`azure\storage\queue\aio\_models.py:70: error: Incompatible types in assignment (expression has type "List[QueueMessage]", variable has type "AsyncIterator[QueueMessage]")  [assignment]`

Reply:
Removing comment as this doesn't even work properly!